### ISSUE TITLE
Lookaround regex not working anymore

### ISSUE THREAD

**Issue by tiagohconte** [alexanderzobnin/grafana-zabbix#1463]
**Describe the bug**
A lookaround regex does not work after updating to last version.

**Expected behavior**
Using the regex `/^Temperatura(?!.*(Gás|Evaporadora))/`
I should receive all the temperature sensors data except from "Gás" and "Evaporadora", but none is shown.

**Screenshots**
![image](https://user-images.githubusercontent.com/54898879/173578293-15bc008e-d2e3-4529-8450-0ba72b90bbde.png)

**Network data**
This is the error that shows up in the query.
```json
error parsing regexp: invalid or unsupported Perl syntax: `(?!`
```
It seems like Go (or Perl) doesn't accept this type of regex, what is weird because this used to work fine in older versions of grafana-zabbix.

**Software versions**

| Grafana | Zabbix | Grafana-Zabbix Plugin |
| ------- | ------ | --------------------- |
| 8.5.5   | 5.4.12  | 4.2.8                 |

**Comment by i-ky** [comment]
Duplicates #1264.

**Comment by eithor** [comment]
@tiagohconte This should now be fixed in 4.2.10, so this issue can be closed.
